src/ui/public/notify/banners/BANNERS.md

```diff
@@ -0,0 +1,148 @@
+# Banners
+
+Use this service to surface banners at the top of the screen. The expectation is that the banner will used an
+`<EuiCallOut />` to render, but that is not a requirement. See [the EUI docs](elastic.github.io/eui/) for more information on toasts and their role within the UI.
```

Comment:
The link to EUI needs to be absolute (`https://elastic.github.io/eui/`) or else GitHub will try to load it relative to the file in the repo.